lib/common/db-renderable-content.js

```diff
@@ -118,6 +120,11 @@ function dbRenderableContentServiceFactory(
 
     DbRenderableContentService.prototype.render = function render (name, options, scope) {
         return this.get(name, scope).then(function (template) {
+            if (!template) {
```

Comment:
Maybe this is way too minuscule of a concern, but wouldn't we want empty strings to be valid templates?